src/ReverseMarkdown/ReverseMarkdown.csproj

```diff
@@ -4,15 +4,12 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <Authors>Babu Annamalai</Authors>
     <TargetFrameworks>net46;netstandard1.6;netstandard2.0</TargetFrameworks>
-    <AssemblyName>ReverseMarkdown</AssemblyName>
```

Comment:
these values default to the project name. so are redundant

Reply:
Agree!